ratelimit/tests.py

```diff
@@ -314,6 +314,15 @@ def test_is_ratelimited_increment(self):
         # Count = 2, 2 > 1.
         assert do_increment(rf.get('/'))
 
+    def test_ip_mask(self):
```

Comment:
can you rewrite this test so that we're actually testing that something gets ratelimited? there are a bunch of tests that follow this pattern, of making up a dummy view and running a couple of fake requests through it. `ip_mask` isn't a public API so we shouldn't test it directly.

this probably needs to be a couple of different tests

Reply:
Updated, please check